lib/octopolo/commands/pull_request.rb

```diff
@@ -3,6 +3,9 @@
   c.desc "Branch to create the pull request against"
   c.flag [:d, :dest, :destination], :arg_name => "destination_branch", :default_value => Octopolo.config.deploy_branch
 
+  c.desc "Pass -x to skip the prompt and infer from branch. Expects JIRA-123_describe_pr"
+  c.switch [:x, :expedite], :arg_name => "expidite"
```

Comment:
"expedite".... spelling error
```suggestion
  c.switch [:x, :expedite], :arg_name => "expedite"
```